src/commands/InitCommand.js

```diff
@@ -131,4 +132,11 @@ export default class InitCommand extends Command {
       FileSystemUtilities.unlinkSync(versionLocation);
     }
   }
+
+  ensurePackagesDir() {
+    if (!this.repository.lernaJson.packages) {
```

Comment:
We don't need to check this, `mkdirp` is just a no-op on directories that already exist.

It's important to run this after `this.ensureLernaJson()` because we need to iterate through `this.repository.packageConfigs` and create every glob parent directory (using [glob-parent](https://www.npmjs.com/package/glob-parent)) present.

```js
// at top of file
import fs from "fs-extra";
import globParent from "glob-parent";

// in method
this.repository.packageConfigs
  .map(pkgGlob => globParent(pkgGlob))
  .forEach(pkgRoot => fs.ensureDirSync(pkgRoot));
```

There's currently no `FileSystemUtilities.mkdirpSync`, and I'd rather not bother for such a one-off. Which makes testing awkward in the current mock-happy situation... ugh